src/store/wikis.js

```diff
@@ -58,10 +58,22 @@ const mutations = {
     const federatedPropertiesSetting = details.public_settings.find(setting => setting.name === 'wikibaseFedPropsEnable')
     const wikibaseFedPropsEnable = federatedPropertiesSetting ? parseInt(federatedPropertiesSetting.value) === 1 : false
 
+    const wwWikibaseStringLengthStringSetting = details.public_settings.find(setting => setting.name === 'wwWikibaseStringLengthString')
```

Comment:
this looks like puke

Reply:
I think at some point we would want to make the API return the default values.
Then most of this puke can go and just use whatever the API would send.
This puts the control of the defaults etc in the API as well which is probably where it belongs.